modules/client/test/integration/client-erc20/methods.test.ts

```diff
@@ -349,6 +377,8 @@ describe("Client ERC20", () => {
           expect(typeof proposal.token.address).toBe("string");
           expect(proposal.token.address).toMatch(/^0x[A-Fa-f0-9]{40}$/i);
         }
+
+        mockedIPFSClient.cat.mockImplementation(defaultCatImplementation);
```

Comment:
What is this for?

Reply:
to revert back to the overwritten implementation on line 343.
This is necessary because we don't mock the graphql calls and therefore don't know beforehand how many calls to `cat` we are going to make

Reply:
Would `mockResolvedValueOnce` do the trick instead?

Reply:
nop as the name suggests it is `once` and graphql can return more than one result